packages/sui-mono/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@s-ui/mono",
-  "version": "1.44.0",
+  "version": "1.44.1-beta.3",
```

Comment:
beta?

Reply:
I'lll removed it before merging. It's in case I change something that needs to be tested :D